<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCompareDisp</class>
 <widget class="QDialog" name="DCompareDisp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare displacements in two plots</string>
  </property>
  <widget class="QLabel" name="hint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This dialog window allows you to compare displacements (depicted by arrows) between two plots currently loaded in the program. Both plots MUST have the same number of atoms and identical positions of atoms in the unrelaxed configuration. The obtained plot will show a difference P1-P2 between the displacements in the first (P1) and the second (P2) plot.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Which components to compare ?</string>
   </property>
   <widget class="QRadioButton" name="compScrew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;screw component</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="compEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;edge component</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Second plot (P2):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="plot2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>First plot (P1):</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="plot1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DCompareDisp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DCompareDisp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
